packages/ses/NEWS.md

```diff
@@ -137,13 +145,17 @@ and subject to breaking changes that will not be signaled by semver.
   gives line-numbers into the generated JavaScript, which often don't match the
   the original lines. This happens even with the normal development-time
   lockdown options setting,
+
   ```js
   errorTaming: 'unsafe'
   ```
+
   or setting the environment variable
+
   ```sh
-  $ export LOCKDOWN_ERROR_TAMING=unsafe
```

Comment:
Was the `$` removal manual or by the .md fixer thing?

I do mildly prefer keeping the `$` prefix so readers can easily this is shell and not js.

Reply:
yep .md fixer https://github.com/DavidAnson/markdownlint/blob/v0.37.4/doc/md014.md

> Rationale: It is easier to copy/paste and less noisy if the dollar signs are omitted when they are not needed. See https://cirosantilli.com/markdown-style-guide#dollar-signs-in-shell-code for more information.

Reply:
> Because some commands do not produce output, it is not a violation if _some_ commands do not have output:

```
$ mkdir test
mkdir: created directory 'test'
$ ls test
```

seems a bit silly that _some_ doesn't include _one_ command with no output 😄